scss/modules/form.scss

```diff
@@ -360,7 +360,7 @@ textarea.ckeditor-preview {
             height: $radioSize;
             padding: 2px;
             border-radius: $radioSize;
-            border: 1px solid $borderColor;
+            border: 1px solid $black;
```

Comment:
the border color is medium gray which should be for all the borders except for them which can be clicked (like the radio-button).